index.js

```diff
@@ -275,13 +269,15 @@ SideMenu.propTypes = {
   autoClosing: PropTypes.bool,
 };
 
+const initialDeviceScreen = Dimensions.get('window');
```

Comment:
You are not using this variable anywhere else but in the `defaultProps`. I'd rather recommend you to use `Dimensions.get('window').width` inside `defaultProps` directly

Reply:
Fixed